tests/test_command_line/test_attack.py

```diff
@@ -20,7 +20,6 @@
             "textattack attack --model cnn-imdb "
             "--attack-from-file tests/sample_inputs/attack_from_file.py^Attack "
             "--num-examples 2  --num-examples-offset 18 --attack-n "
-            "--shuffle=False"
```

Comment:
Why do we disable shuffle? Do we always perform tests on a fixed sentences of the dataset? Isn't it better to test from random sentences everytime?
Make testing deterministic(by fixed) vs improving coverage (by random)